examples/hcl/ubuntu/build.pkr.hcl

```diff
@@ -0,0 +1,34 @@
+
+build {
+  name = "ubuntu"
+  description = <<EOF
+This build creates ubuntu images for ubuntu versions :
+* 16.04
+* 18.04 
+For the following builers :
+* virtualbox-iso
+EOF
+
+  source "source.virtualbox-iso.base-ubuntu-amd64" {
+    iso_url                 = local.iso_url_ubuntu_1604
+    iso_checksum            = "file:${local.iso_checksum_url_ubuntu_1604}"
+    output_directory        = "ubuntu_1604_amd64"
+    boot_command            = local.ubuntu_1604_boot_command
+    boot_wait               = "10s"
+  }
+
+  source "source.virtualbox-iso.base-ubuntu-amd64" {
+    iso_url                 = local.iso_url_ubuntu_1804
+    iso_checksum            = "file:${local.iso_checksum_url_ubuntu_1804}"
+    output_directory        = "ubuntu_1804_amd64"
+    boot_command            = local.ubuntu_1804_boot_command
+    boot_wait               = "5s"
+  }
```

Comment:
This is really nice but both builds will be named `ubuntu.virtualbox-iso.base-ubuntu-amd64` and therefore you cannot only build ubuntu 1804 or ubuntu 1604, which I think would be nice to have here.

Reply:
Ah never-mind I added the possibility to name source there